offlinedocs/pages/[[...slug]].tsx

```diff
@@ -27,7 +27,7 @@ import {
 } from "@chakra-ui/react";
 import fm from "front-matter";
 import { readFileSync } from "fs";
-import _ from "lodash";
+import _, { min } from "lodash";
```

Comment:
```suggestion
import _ from "lodash";
```

linter doesn't seem to be calling this out, but this is unused